app/jobs/create_palette_job.rb

```diff
@@ -5,9 +5,11 @@
 class CreatePaletteJob < ApplicationJob
   queue_as :default
 
-  def perform(collection)
+  def perform(palette_params, collection)
```

Comment:
Before, we simply didn't pass any palette_params. We were going entirely with default. I added these in now since we need to re-calculate the palette based on the params from the posterize chunk of the manual clean form.